plugins/inputs/win_services/win_services_test.go

```diff
@@ -0,0 +1,101 @@
+// +build windows
+
+//this test must be run under administrator account
+package win_services
+
+import (
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+    "golang.org/x/sys/windows/svc/mgr"
+)
+
+var InvalidServices = []string{"XYZ1@", "ZYZ@", "SDF_@#"}
+var KnownServices = []string{"LanmanServer", "TermService"}
+
+func TestList(t *testing.T) {
+	services, err := listServices(KnownServices)
```

Comment:
We don't want to add any tests that require special permissions or certain services to running (unless it can also set them up). It would be better to use an interface with a test version.